postcss.config.js

```diff
@@ -1,7 +1,8 @@
+import process from 'process'
+
 export default {
   plugins: {
-    tailwindcss: {},
-    autoprefixer: {},
+    '@tailwindcss/postcss': {},
```

Comment:
We could have opted for TW's vite plugin, but for now I kept things the way they were. 

https://tailwindcss.com/docs/installation/using-vite

Wasn't sure if it would bypass all the other postcss features we use.